public/app/config/channels.js

```diff
@@ -6,4 +6,5 @@ module.exports = {
   SHOW_LOAD_DATASET_PROMPT_CHANNEL: 'prompt:dataset:load:show',
   RESPOND_LOAD_DATASET_PROMPT_CHANNEL: 'prompt:dataset:load:respond',
   LOAD_DATASET: 'dataset:load',
+  EXECUTE_PYTHON_SCRIPT: 'script:python',
```

Comment:
I would go for `script:python:execute` as we are going to have other operations such as kill, pause, etc.